lessons/basics/collections.md

```diff
@@ -40,6 +40,9 @@ iex> [1, 2] ++ [3, 4, 1]
 [1, 2, 3, 4, 1]
 ```
 
+A side note about the name (`++/2`) format used above: In Elixir (and Erlang, upon which Elixir is built), a function or operator name has two components: the name you give it (here `++`) and the number of arguments it takes (two in this case), combined with a slash. (The number of arguments is often referred to as "arity.") We'll talk more about this later; this knowledge will help you understand the notation for now.
```

Comment:
I personally think this should emphasize the word `arity` in priority over explaining it in plain terms. The reasoning being that arity is core to the entirety of notation in Elixir/Erlang and it will not be going anywhere. I think this implies that moreso than the way it reads now. I think how I would word it is:

```
A side note about the name (`++/2`) format used above: In Elixir (and Erlang, upon which Elixir is built), a function or operator name has two components: the name you give it (here `++`) and its `arity`. Arity is a core part of speaking about Elixir (and Erlang) code. It is the number of arguments a given function takes (two in this case), combined with a slash. We'll talk more about this later; this knowledge will help you understand the notation for now.
```